src/main/js/wallet/MoneroWalletFull.js

```diff
@@ -86,19 +85,19 @@ class MoneroWalletFull extends MoneroWalletKeys {
    * @param {string} configOrPath.password - password of the wallet to open
    * @param {string|number} configOrPath.networkType - network type of the wallet to open (one of "mainnet", "testnet", "stagenet" or MoneroNetworkType.MAINNET|TESTNET|STAGENET)
    * @param {Uint8Array} configOrPath.keysData - wallet keys data to open (optional if path provided)
-   * @param {Uint8Array} configOrPath.cacheData - wallet cache data to open (optional)
-   * @param {string} configOrPath.serverUri - uri of the wallet's daemon (optional)
-   * @param {string} configOrPath.serverUsername - username to authenticate with the daemon (optional)
-   * @param {string} configOrPath.serverPassword - password to authenticate with the daemon (optional)
-   * @param {boolean} configOrPath.rejectUnauthorized - reject self-signed server certificates if true (default true)
-   * @param {MoneroRpcConnection|object} configOrPath.server - MoneroRpcConnection or equivalent JS object configuring the daemon connection (optional)
-   * @param {boolean} configOrPath.proxyToWorker - proxies wallet operations to a worker in order to not block the main thread (default true)
-   * @param {fs} configOrPath.fs - Node.js compatible file system to use (defaults to disk or in-memory FS if browser)
+   * @param {Uint8Array} [configOrPath.cacheData] - wallet cache data to open (optional)
+   * @param {string} [configOrPath.serverUri] - uri of the wallet's daemon (optional)
+   * @param {string} [configOrPath.serverUsername] - username to authenticate with the daemon (optional)
+   * @param {string} [configOrPath.serverPassword] - password to authenticate with the daemon (optional)
+   * @param {boolean} [configOrPath.rejectUnauthorized] - reject self-signed server certificates if true (default true)
+   * @param {MoneroRpcConnection|object} [configOrPath.server] - MoneroRpcConnection or equivalent JS object configuring the daemon connection (optional)
+   * @param {boolean} [configOrPath.proxyToWorker] - proxies wallet operations to a worker in order to not block the main thread (default true)
+   * @param {fs} [configOrPath.fs] - Node.js compatible file system to use (defaults to disk or in-memory FS if browser)
    * @param {string} password - password of the wallet to open
```

Comment:
These are actually optional too since they might only use a single config param. I can do this cleanup though. More important to establish functionality now.